src/main/java/com/meilisearch/sdk/Dump.java

```diff
@@ -0,0 +1,59 @@
+package com.meilisearch.sdk;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import com.meilisearch.sdk.exceptions.MeiliSearchApiException;
+
+/**
+ * Wrapper around the MeilisearchHttpRequest class to ease usage for Meilisearch dumps
+ */
+public class Dump {
+	private MeiliSearchHttpRequest meiliSearchHttpRequest;
+	private String status;
+	private String uid;
+
+	/**
+	 * Creates and sets up an instance of Dump to simplify Meilisearch API calls to manage dumps
+	 *
+	 * @param config Meilisearch configuration
+	 */
+	public Dump(Config config) { this.meiliSearchHttpRequest = new MeiliSearchHttpRequest(config); }
+
+	/**
+	 * Creates a dump
+	 * Refer https://docs.meilisearch.com/references/dump.html#create-a-dump
+	 *
+	 * @return Dump object with Meilisearch API response
+	 * @throws Exception if something goes wrong
+	 */
+	public Dump createDump() throws Exception, MeiliSearchApiException {
+		Dump dump = new Gson().fromJson(
+			this.meiliSearchHttpRequest.post("/dumps", ""),
+			Dump.class
+		);
+
+		return dump;
```

Comment:
```suggestion
		return new Gson().fromJson(
			this.meiliSearchHttpRequest.post("/dumps", ""),
			Dump.class
		);
```

Reply:
As a suggestion: you can directly return the value. If you are not manipulating it in any way, there is no need to store it in a variable :)